gpt4all-chat/responsetext.cpp

```diff
@@ -570,14 +569,15 @@ void ResponseText::handleCodeBlocks()
         QRegularExpressionMatch match = reWhitespace.match(capturedText);
         if (match.hasMatch()) {
             const QString firstWord = match.captured(1).trimmed();
-            if (firstWord == "python"
-                || firstWord == "cpp"
-                || firstWord == "c++"
+            if (firstWord == "bash"
                 || firstWord == "c"
-                || firstWord == "bash"
+                || firstWord == "c++"
+                || firstWord == "cpp"
+                || firstWord == "java"
                 || firstWord == "javascript"
-                || firstWord == "typescript"
-                || firstWord == "java") {
+                || firstWord == "php"
```

Comment:
You should remove this hunk as I'm guessing you haven't added php syntax in this patch?